Sprint 1

Sapr0nov opened the pull request:
Исправлены замечания к первому реквесту, 
прим. модуль parcel-transformer-templator - подключаемый трансоформер для парсера (самописный шаблонизатор для файлов jssx).

Sapr0nov committed:
structure

Sapr0nov committed:
fix readme

Sapr0nov committed:
Create README.md

Sapr0nov committed:
config postCSS

Sapr0nov committed:
prepare

Sapr0nov committed:
modified:   .parcelrc
	new file:   .posthtmlrc
	modified:   README.md
	modified:   package-lock.json
	modified:   package.json
	deleted:    src/components/button/button.js
	modified:   src/components/button/button.jssx
	modified:   src/components/button/index.js
	modified:   src/config/index.css
	renamed:    src/style.css -> src/index.css
	deleted:    src/index.html
	new file:   src/index.js
	new file:   src/index.jssx
	deleted:    src/main.js
	modified:   src/utils/templator.js

Sapr0nov committed:
deleted:    transformer-templator/LICENSE
	deleted:    transformer-templator/Read.me
	deleted:    transformer-templator/lib/transformer-templator.js
	deleted:    transformer-templator/package-lock.json
	deleted:    transformer-templator/package.json

Sapr0nov committed:
modified:   .parcelrc
	modified:   package-lock.json
	modified:   package.json
	new file:   src/components/button/button.html
	deleted:    src/components/button/button.jssx
	modified:   src/index.js
	modified:   src/index.jssx

Sapr0nov committed:
modified:   package-lock.json
	modified:   package.json

Sapr0nov committed:
renamed:    src/pages/about/about.css -> src/pages/404/404.css
	renamed:    src/pages/about/about.js -> src/pages/404/404.html
	renamed:    src/pages/about/about.tmpl.js -> src/pages/404/404.js
	renamed:    src/pages/about/index.js -> src/pages/500/500.css
	renamed:    src/pages/home/home.css -> src/pages/500/500.html
	renamed:    src/pages/home/home.js -> src/pages/500/500.js
	deleted:    src/pages/home/home.tmpl.js
	deleted:    src/pages/home/modules/logout/index.js
	renamed:    src/pages/home/index.js -> src/pages/index/modules/login/index.js
	renamed:    src/pages/home/modules/login/login.css -> src/pages/index/modules/login/login.css
	renamed:    src/pages/home/modules/login/login.js -> src/pages/index/modules/login/login.js
	renamed:    src/pages/home/modules/login/login.tmpl.js -> src/pages/index/modules/login/login.tmpl.js
	renamed:    src/pages/home/modules/login/index.js -> src/pages/index/modules/logout/index.js
	renamed:    src/pages/home/modules/logout/logout.css -> src/pages/index/modules/logout/logout.css
	renamed:    src/pages/home/modules/logout/logout.js -> src/pages/index/modules/logout/logout.js
	renamed:    src/pages/home/modules/logout/logout.tmpl.js -> src/pages/index/modules/logout/logout.tmpl.js
	deleted:    src/utils/templator.js
	new file:   static/404error.png
	new file:   static/500error.png

Sapr0nov committed:
modified:   package-lock.json
	modified:   package.json
	modified:   src/index.jssx
	renamed:    src/modules/chats/components/message/index.js -> src/modules/chats/components/message/message.html
	deleted:    src/modules/chats/components/message/message.tmpl.js

Sapr0nov committed:
Merge branch 'sprint_1' of https://github.com/Bezumstvo/middle.messenger.praktikum.yandex into sprint_1

Sapr0nov committed:
modified:   src/index.js
	modified:   src/index.jssx
	modified:   src/pages/404/404.css
	modified:   src/pages/404/404.html
	deleted:    src/pages/404/404.js

Sapr0nov committed:
Merge branch 'sprint_1' of https://github.com/Bezumstvo/middle.messenger.praktikum.yandex into sprint_1

Sapr0nov committed:
modified:   package-lock.json
	modified:   package.json
	modified:   src/index.js
	new file:   src/srv.js

Sapr0nov committed:
modified:   package.json
	deleted:    src/config/index.css
	deleted:    src/index.js
	deleted:    src/layout/main/main.css
	deleted:    src/layout/main/main.js
	deleted:    src/layout/main/main.tmpl.js
	deleted:    src/pages/500/500.js
	renamed:    src/index.css -> src/pages/index/index.css
	renamed:    src/layout/main/index.js -> src/pages/index/index.js
	renamed:    src/index.jssx -> src/pages/index/index.jssx
	modified:   src/srv.js

Sapr0nov committed:
404 page finished

Sapr0nov committed:
page 500

Sapr0nov committed:
deleted:    src/components/button/button.css
	deleted:    src/components/button/button.html
	new file:   src/components/form_auth/form_auth.css
	new file:   src/components/form_auth/form_auth.html
	renamed:    src/components/button/index.js -> src/components/form_auth/form_auth.js
	modified:   src/pages/index/index.jssx

Sapr0nov committed:
modified:   src/components/form_auth/form_auth.css
	modified:   src/components/form_auth/form_auth.html
	new file:   src/components/form_registration/form_registration.css
	new file:   src/components/form_registration/form_registration.html
	new file:   src/components/form_registration/form_registration.js
	modified:   src/pages/index/index.jssx

Sapr0nov committed:
modified:   package-lock.json
	modified:   package.json
	renamed:    src/modules/chats/chats.css -> src/components/chat_preview/chat_preview.css
	new file:   src/components/chat_preview/chat_preview.html
	deleted:    src/modules/chats/chats.js
	deleted:    src/modules/chats/chats.tmpl.js
	modified:   src/pages/index/index.jssx

Sapr0nov committed:
modified:   package-lock.json
	modified:   package.json
	modified:   src/components/chat_preview/chat_preview.css
	deleted:    src/components/chat_preview/chat_preview.html
	new file:   src/components/chat_preview/chat_preview.js
	new file:   src/components/chat_preview/chat_preview.jssx
	modified:   src/pages/index/index.jssx

Sapr0nov committed:
readme

Sapr0nov committed:
modified:   README.md

Sapr0nov committed:
modified:   README.md

Sapr0nov committed:
modified:   README.md

Sapr0nov committed:
modified:   README.md

Sapr0nov committed:
modified:   src/pages/404/404.html
	modified:   src/components/chat_preview/chat_preview.css
	modified:   src/pages/500/500.html
	deleted:    static/500error.png
	deleted:    static/icon.png
	deleted:    static/search.svg
	src/components/chat_preview/search.svg
	src/pages/500/500error.png

Sapr0nov committed:
modified:   src/components/chat_preview/chat_preview.css
	renamed:    static/search.svg -> src/components/chat_preview/search.svg
	modified:   src/pages/500/500.html
	renamed:    static/500error.png -> src/pages/500/500error.png
	deleted:    static/icon.png

Sapr0nov committed:
modified:   src/components/chat_preview/chat_preview.css
	new file:   src/components/messages/message.css
	new file:   src/components/messages/message.html
	new file:   src/components/messages/message.js
	modified:   src/pages/404/404.css
	modified:   src/components/chat_preview/chat_preview.css
	modified:   src/components/chat_preview/chat_preview.jssx
	modified:   src/components/form_auth/form_auth.css
	modified:   src/components/form_auth/form_auth.js
	modified:   src/components/form_registration/form_registration.css
	deleted:    src/components/messages/message.css
	deleted:    src/components/messages/message.html
	deleted:    src/components/messages/message.js
	deleted:    src/modules/chats/components/message/message.css
	deleted:    src/modules/chats/components/message/message.html
	deleted:    src/modules/chats/components/message/message.js
	deleted:    src/modules/chats/index.js
	deleted:    src/modules/chats/utils/getFullname.js
	deleted:    src/modules/chats/utils/getIsOnline.js
	modified:   src/pages/index/index.css
	modified:   src/pages/index/index.js
	modified:   src/pages/index/index.jssx
	deleted:    src/pages/index/modules/login/index.js
	deleted:    src/pages/index/modules/login/login.css
	deleted:    src/pages/index/modules/login/login.js
	deleted:    src/pages/index/modules/login/login.tmpl.js
	deleted:    src/pages/index/modules/logout/index.js
	deleted:    src/pages/index/modules/logout/logout.css
	deleted:    src/pages/index/modules/logout/logout.js
	deleted:    src/pages/index/modules/logout/logout.tmpl.js

Sapr0nov committed:
modified:   src/components/chat_preview/chat_preview.css
	modified:   src/components/chat_preview/chat_preview.jssx
	modified:   src/components/form_auth/form_auth.css
	modified:   src/components/form_auth/form_auth.js
	modified:   src/components/form_registration/form_registration.css
	new file:   src/components/message/attach.svg
	new file:   src/components/message/menu.svg
	new file:   src/components/message/message.css
	new file:   src/components/message/message.html
	renamed:    src/components/messages/message.js -> src/components/message/message.js
	new file:   src/components/message/sendBtn.svg
	deleted:    src/components/messages/message.css
	deleted:    src/components/messages/message.html
	deleted:    src/modules/chats/components/message/message.css
	deleted:    src/modules/chats/components/message/message.html
	deleted:    src/modules/chats/components/message/message.js
	deleted:    src/modules/chats/index.js
	deleted:    src/modules/chats/utils/getFullname.js
	deleted:    src/modules/chats/utils/getIsOnline.js
	modified:   src/pages/index/index.css
	modified:   src/pages/index/index.js
	modified:   src/pages/index/index.jssx
	deleted:    src/pages/index/modules/login/index.js
	deleted:    src/pages/index/modules/login/login.css
	deleted:    src/pages/index/modules/login/login.js
	deleted:    src/pages/index/modules/login/login.tmpl.js
	deleted:    src/pages/index/modules/logout/index.js
	deleted:    src/pages/index/modules/logout/logout.css
	deleted:    src/pages/index/modules/logout/logout.js
	deleted:    src/pages/index/modules/logout/logout.tmpl.js

Sapr0nov committed:
modified:   src/components/chat_preview/chat_preview.css
	renamed:    src/components/form_auth/form_auth.css -> src/components/form_auth/form-auth.css
	modified:   src/components/form_auth/form_auth.html
	modified:   src/components/form_auth/form_auth.js
	modified:   src/components/form_registration/form_registration.css
	modified:   src/components/form_registration/form_registration.html
	modified:   src/components/form_registration/form_registration.js
	modified:   src/components/message/message.css
	renamed:    src/utils/isEmpty.js -> src/components/settings/settings.css
	new file:   src/components/settings/settings.html
	new file:   src/components/settings/settings.js
	modified:   src/pages/index/index.css
	modified:   src/pages/index/index.js
	modified:   src/pages/index/index.jssx
	new file:   src/utils/Cookies.js

Sapr0nov committed:
final version

Sapr0nov committed:
modified:   src/components/settings/settings.css
	modified:   src/components/settings/settings.html

Sapr0nov committed:
update package

Sapr0nov committed:
modified:   src/components/settings/settings.html
	modified:   src/pages/index/index.jssx

Sapr0nov committed:
add read.me

Sapr0nov committed:
fixed tips

dmitrymorozoff commented on package.json:
```diff
@@ -0,0 +1,36 @@
+  "scripts": {
+    "start": "node src/srv.js",
+    "clean": "rimraf .parcel-cache rimraf .cache && rimraf dist && mkdir dist",
+    "dev": "parcel src/pages/index/index.jssx",
```
Здесь, прямо каждую отдельную страницу собирать отдельно не нужно

dmitrymorozoff commented on src/components/form_registration/form_registration.html:
```diff
@@ -0,0 +1,34 @@
+<form id="reg" class="form form-reg">
```
А почему, где-то используется шаблонизатор, а где то нет. Лучше было бы работать со всеми файлами в едином формате

dmitrymorozoff commented on src/components/glitchEffect/glitchEffect.css:
```diff
@@ -0,0 +1,272 @@
+.glitch, .glow {
```
Везде файлы именуются в формате с нижним подчеркиванием. А этот файл в формате camelCase. Нужно привести именование к единому формату

dmitrymorozoff commented on src/components/chat/chat.html:
```diff
@@ -0,0 +1,18 @@
+<div class="chat">
```
Все картинки, лучше поместить в одну папку, например images. Или вообще вынести их в корень проекта в папку static

dmitrymorozoff reviewed (commented):
Спасибо за внесенные правки. Работу приму.

Оставил еще немного комментариев по улучшению работы. Так же не стоит закрывать пулл реквест, пока работа не принята. Можно просто пушить изменения в том же самом реквесте.

Sapr0nov committed:
modified:   package.json
	modified:   src/components/chat/chat.css
	renamed:    src/components/chat/chat.html -> src/components/chat/chat.jssx
	renamed:    src/components/form_auth/form_auth.html -> src/components/form_auth/form_auth.jssx
	renamed:    src/components/form_registration/form_registration.html -> src/components/form_registration/form_registration.jssx
	renamed:    src/components/glitchEffect/glitchEffect.css -> src/components/glitch_effect/glitch_effect.css
	renamed:    src/components/glitchEffect/glitchEffect.html -> src/components/glitch_effect/glitch_effect.html
	modified:   src/components/settings/settings.css
	modified:   src/components/settings/settings.jssx
	modified:   src/components/side_bar/side_bar.css
	modified:   src/pages/404/404.jssx
	renamed:    src/pages/500/500.html -> src/pages/500/500.jssx
	renamed:    src/components/settings/arrow_back.svg -> static/arrow_back.svg
	renamed:    src/components/chat/attach.svg -> static/attach.svg
	renamed:    src/components/chat/menu.svg -> static/menu.svg
	renamed:    src/components/settings/placeholder.png -> static/placeholder.png
	renamed:    src/components/side_bar/search.svg -> static/search.svg
	renamed:    src/components/chat/sendBtn.svg -> static/sendBtn.svg

Sapr0nov committed:
Merge branch 'main' into sprint_1
